packages/trpc/server/routers/viewer/slots.tsx

```diff
@@ -102,8 +106,9 @@ export const slotsRouter = createRouter().query("getSchedule", {
 export async function getSchedule(
   input: {
```

Comment:
Don't Repeat Yourself
```suggestion
  input: z.infer<typeof getScheduleSchema>
```

Reply:
Oh right, this is definitely better! 👍